oat-sa/extension-tao-testqti
# [TAO-8311] Update package.json

Related to : https://oat-sa.atlassian.net/browse/TAO-8311

Retry to sync data if previous request failed due to connectivity problem

How to test:
- Prepare a TAO instance with NFER extension
- Import LCL Pilot Final test from nferlightweight env
- Run delivery on iOS 9.3.5
- Check there is no connectivity error interruption during test execution if device is connected to network
